packages/app/documents/service.ts

```diff
@@ -92,7 +92,7 @@ export async function createDocument(
         const [last] = insertedDocument['x-meditor'].states.slice(-1)
         const targetState = last.target
 
-        await safelyPublishDocumentChangeToQueue(
+        safelyPublishDocumentChangeToQueue(
```

Comment:
Both the changes in this file are enacting the following idea (push back if it's not wanted): we shouldn't await starting these document publishes since our distributed publishing is like a side effect fired when creating a document. If it fails, we want to know (we can see that in the UI and logs), but we don't want to hold up saving the document locally if something has happened in our `lib/nats` dependency.